.devcontainer/test-setup.sh

```
@@ -0,0 +1,52 @@
+#!/bin/bash
+set -e
+
+echo "🧪 Testing GitHub Codespaces setup..."
+
+# Test PostgreSQL connection
+echo -n "Testing PostgreSQL... "
+if PGPASSWORD=${POSTGRES_PASSWORD:-apppass} psql -h ${POSTGRES_HOST:-db} -U ${POSTGRES_USER:-appuser} -d ${POSTGRES_DB:-appdb} -c '\q' 2>/dev/null; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)